apps/omg/test/omg/fees_test.exs

```diff
@@ -111,20 +117,35 @@ defmodule OMG.FeesTest do
 
       assert Fees.for_transaction(transaction, @fees) == :no_fees_required
     end
-  end
 
-  describe "filter_fees/2" do
-    test "does not filter when an empty list is passed" do
-      assert Fees.filter_fees(@fees, []) == {:ok, @fees}
-    end
+    test "returns an empty hash when given an unsuported tx type" do
+      transaction = %OMG.State.Transaction.Recovered{
+        signed_tx: %OMG.State.Transaction.Signed{raw_tx: %OMG.TransactionHelper.Dummy{}, sigs: []},
```

Comment:
I see that `OMG.TransactionHelper.Dummy` is only used here. I would vote for defining it inside the testing module so that the test is somewhat independent